servers/rendering/rasterizer.h

```diff
@@ -231,7 +231,7 @@ class RasterizerScene {
 
 	virtual RID light_instance_create(RID p_light) = 0;
 	virtual void light_instance_set_transform(RID p_light_instance, const Transform &p_transform) = 0;
-	virtual void light_instance_set_shadow_transform(RID p_light_instance, const CameraMatrix &p_projection, const Transform &p_transform, float p_far, float p_split, int p_pass, float p_shadow_texel_size, float p_bias_scale = 1.0) = 0;
+	virtual void light_instance_set_shadow_transform(RID p_light_instance, const CameraMatrix &p_projection, const Transform &p_transform, float p_far, float p_split, int p_pass, float p_shadow_texel_size, float p_bias_scale = 1.0, float p_range_begin = 0, const Vector2 &p_uv_scale = Vector2()) = 0;
```

Comment:
This also needs to be changed in `drivers/dummy/rasterizer_dummy.h` (currently used by the javascript platform).

Reply:
I would not rather have to update that myself for a while, since the APIs are changing too much and keeping that one up to date is quite a hassle.

Reply:
Well it's fairly easy to keep the prototypes in sync...

But well, the javascript platform no longer builds since the DisplayServer merge, so I guess it's not critical to fix that dummy rasterizer now.